dev/Breeze/conftest.py

```diff
@@ -0,0 +1,23 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+import sys
+from pathlib import Path
+
+ROOT_DIR = str(Path(__file__))
+
+sys.path.insert(1, ROOT_DIR)
```

Comment:
This would also not be needed is Breeze is an installable package!

Reply:
This is for now only for testing which we want to do during the CI builds, so this is needed as those sources are in the same monorepo (which I think is something we should keeep for as long as humanly possible).  But we would not include the file in the package even if we decide to have the package (and we do not need to run tests when it is released as package).